src/webgpu/api/operation/rendering/stencil.spec.ts

```diff
@@ -222,9 +281,84 @@ g.test('stencil_compare_func')
   )
   .fn(async t => {
     const { stencilCompare, stencilRefValue, _expectedColor } = t.params;
+
     t.checkStencilCompareFunction(stencilCompare, stencilRefValue, _expectedColor);
   });
 
+g.test('stencil_pass_operation')
+  .desc(
+    `
+    Test that each stencil operation works with the given stencil values correctly as expected.
+  `
+  )
+  .params(u =>
+    u //
+      .combineWithParams([
+        { stencilOperation: 'keep', initialStencil: 1, referenceStencil: 3, expectedStencil: 1 },
+        { stencilOperation: 'zero', initialStencil: 1, referenceStencil: 3, expectedStencil: 0 },
+        { stencilOperation: 'replace', initialStencil: 1, referenceStencil: 3, expectedStencil: 3 },
+        {
+          stencilOperation: 'invert',
```

Comment:
ok, I rename `stencilOperation` to `passOp` in the test, then rename the test to `stencil_passOp_operation` with a TODO. But, I keep `checkStencilOperation` function name because it will be able to be used for `failOp` and 'depthFailOp` later.